packages/contact-center/user-state/src/helper.ts

```diff
@@ -15,8 +17,23 @@ export const useUserState = ({idleCodes, agentId, cc}) => {
       setElapsedTime(prevTime => prevTime + 1);
     }, 1000);
 
+    const handleStateChange = (data) => {
+      if (data && typeof data === 'object' && data.type === 'AgentStateChangeSuccess') {
```

Comment:
Why is `typeof data` required? All data are objects no?

Reply:
This is just to ensure that `data.type` doesn't throw an error just in case the data received is not an object